src/main/target/common_fc_pre.h

```diff
@@ -51,7 +51,6 @@
 #define USE_GYRO_DATA_ANALYSE
 #define USE_ADC
 #define USE_ADC_INTERNAL
-#define USB_CDC_HID
```

Comment:
Thanks for the quick fix, @conkerkh. This looks like it also disables HID for all targets that do not explicitly enable it, which is not what we want for a permanent solution (and before the permanent solution we don't want it so that users can test / provide feedback). What is your take on making this configurable at runtime?

Reply:
I like this idea. I can do PG for this.

Reply:
Thanks.

Reply:
Added in #5525